api/proto/config/devconfig.proto

```diff
@@ -62,6 +62,9 @@ message EdgeDevConfig {
   // containing certificate and other details, to be
   // used for sensitive data decryption
   repeated CipherContext cipherContexts = 19;
+
+  repeated ContentTree contentInfo = 20;
+  repeated Volume volumes = 21;
```

Comment:
Are all volumes used by Apps guaranteed to be part of this Array?
And Any content referenced by the Volumes guaranteed to be part of ContentTree?
If not - can these be treated as errors?

Reply:
And what is the behavior Expected here?

Device will retain the volume ( irrespective of refcount ) as long as it is present in config?
If volume is not present in DevConfig - it will be deleted once refcount for the volume goes to zero?

Same with ContentTree?

Reply:
@kalyan-nidumolu in general the controller is expected to give consistent and self-sufficient configuration to EVE. Any inconsistencies are to be treated as errors. And this is not specific to volumes. Also, w.r.t what fields can change, that is an implementation level discussion (not API level). Please provide any feedback you may have here: https://wiki.lfedge.org/display/EVE/Volume+%28and+Content%29+device+API